fix: add path for suggested perf counter

jpoon opened the pull request:
The name and path for the perf counter are not the same. Tbh, I'm not entirely certain if the original perf counter mentioned `Avg. Transaction ms/Commit` is the same as `\Service Fabric Replicator()\Avg. Commit ms/Operation`

On an ASF node:
```
PS C:\Users\adminuser> (Get-Counter -ListSet "Service Fabric Replicator").Paths
\Service Fabric Replicator()\Enqueued Bytes/Sec
\Service Fabric Replicator()\Enqueued Operations/Sec
\Service Fabric Replicator()\Current Role
\Service Fabric Replicator()\% Replication Queue Usage
\Service Fabric Replicator()\Avg. Cleanup ms/Operation
\Service Fabric Replicator()\Avg. Complete ms/Operation
\Service Fabric Replicator()\Avg. Commit ms/Operation
\Service Fabric Replicator()# Operations Replication Queue
\Service Fabric Replicator(*)# Bytes Replication Queue
```

azurecla commented:
Hi __@SunBuild__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sumuth). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jpoon committed:
fix: add path for suggested perf counter

The name and path for the perf counter are not the same. 

```
PS C:\Users\adminuser> (Get-Counter -ListSet "Service Fabric Replicator").Paths
\Service Fabric Replicator()\Enqueued Bytes/Sec
\Service Fabric Replicator()\Enqueued Operations/Sec
\Service Fabric Replicator()\Current Role
\Service Fabric Replicator()\% Replication Queue Usage
\Service Fabric Replicator()\Avg. Cleanup ms/Operation
\Service Fabric Replicator()\Avg. Complete ms/Operation
\Service Fabric Replicator()\Avg. Commit ms/Operation
\Service Fabric Replicator()# Operations Replication Queue
\Service Fabric Replicator(*)# Bytes Replication Queue
```
